.swiftformat

```diff
@@ -7,9 +7,13 @@
 --disable hoistAwait
 --disable hoistTry
 --disable redundantReturn
+--disable hoistPatternLet
+--disable preferForLoop
```

Comment:
forEach i think is the more modern swift way, anything against? @pepicrft

Reply:
99% of the time I prefer standard for loop

Reply:
https://stackoverflow.com/questions/45333177/when-to-use-foreach-instead-of-for-in
describes very well the benefit for .forEach especially in functional programming

Reply:
We can discuss it separately, but let's keep it as it is in the scope of this PR

Reply:
I agree with @danyf90. That's a different discussion to have. I'd rather not spend energy discussion stylistics preferences unless it's clearly proven that a listing decision has a negative impact on the experience of contributing to the repository, which I don't believe is the case here.

Reply:
the fun fact: there are already quite a lot of `.forEach {`
https://github.com/search?q=repo%3Atuist%2Ftuist%20.forEach%20%7B&type=code